src/goto-instrument/nondet_static.cpp

```diff
@@ -21,6 +21,14 @@ Date: November 2011
 
 #include "nondet_static.h"
 
+static bool
```

Comment:
Nice 👍 but perhaps rather than making this static and hiding it away in here, would it be better to treat this as a proper utility function and move into one of the `util/*` files? Perhaps `util/types.cpp/.h` might be a good location as there is already an `is_number` utility function in there. Happy if you think there is a better candidate file to move it to though.